SKILL.md

```
@@ -1,68 +1,236 @@
+- `~/.openclaw/agents/.../*.jsonl` — Session transcripts (for memory capture)
+- `~/.nima/` — Local memory database (SQLite or LadybugDB)
+- `~/.openclaw/extensions/` — Hook installation
+
+### Network Calls
+Embeddings are sent to external APIs:
+- **Voyage AI** (`api.voyageai.com`) — Default embedding provider
+- **OpenAI** (`api.openai.com`) — Optional embedding provider
+- **Local** — No external calls when using sentence-transformers
+
+### Required Environment Variables
+| Variable | Purpose | Required |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)